src/components/Book.js

```diff
@@ -10,23 +12,48 @@ function Book(props) {
   const removeNow = (m) => {
     dispatch(deleteBook(m));
   };
+  const min = 1;
+  const max = 100;
+  const value = Math.floor(Math.random() * (max - min + 1) + min);
 
   return (
     <>
       {mapData.map((i) => (
         <div key={i.item_id} className="flex justify-between border border-#94a3b8 m-10 sm:mx-20 sm:my-10 p-6 bg-white">
           <div className="flex flex-col gap-7">
             <div>
-              <h3 className="text-base font-semibold text-slate-500 ">{i.genre}</h3>
+              <h3 className="text-base font-semibold text-slate-500 ju ">{i.category}</h3>
```

Comment:
- [ ] Kindly stick with the design colors for the color of the logo and the background color of the `add book` button with Zeplin guidelines.

| Your design | Zeplin design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/191786706-9da86919-e13f-4833-81c3-e4f4e07df64b.png) | ![image](https://user-images.githubusercontent.com/24703171/191786774-cfdb3662-1c87-43aa-9f85-ea28446c3abf.png) |
| ![image](https://user-images.githubusercontent.com/24703171/191786919-6fb6ae54-77fc-4b42-8f2f-bf969ad39140.png) | ![image](https://user-images.githubusercontent.com/24703171/191787013-db4831dd-f556-479c-9214-c4c580493e67.png) |